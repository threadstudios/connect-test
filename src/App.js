import Inferno from 'inferno';
import Component from 'inferno-component';
import VehicleList from './components/VehicleList';

class App extends Component {
    componentDidMount() {
        // set SSR state
        this.setState(__PROPS__);
    }
    render() {
        return (
            <section id="app">
                <VehicleList {...this.state} />
            </section>
        );
    }
}

export default App;