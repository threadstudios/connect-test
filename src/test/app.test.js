import Inferno from 'inferno';
import { renderToSnapshot } from 'inferno-test-utils';
import App from '../App';

describe("The core App Component", () => {

    window.__PROPS__ = {}

    it("should render without an error", () => {
        expect(renderToSnapshot(<App />)).toMatchSnapshot();
    })

})