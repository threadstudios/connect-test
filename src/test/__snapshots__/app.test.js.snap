// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The core App Component should render without an error 1`] = `<section />`;
